apps/api/src/app/routes/__chainId/tokens/__tokenAddress/usdPrice.ts

```diff
@@ -69,7 +72,10 @@ const root: FastifyPluginAsync = async (fastify): Promise<void> => {
     async function (request, reply) {
       const { chainId, tokenAddress } = request.params;
 
-      const price = await usdService.getUsdPrice(chainId, tokenAddress);
+      const price = await usdService.getUsdPrice(
+        chainId,
+        tokenAddress === '-' ? undefined : tokenAddress
```

Comment:
Maybe this needs a comment? also giving `-` a meaningful constant name?

Reply:
Fixed: https://github.com/cowprotocol/bff/pull/137/commits/12d806f41cd4a7f3bb4734f38aa5f56fcf8c4496